gwcs/wcs.py

```diff
@@ -258,15 +258,20 @@ def set_transform(self, from_frame, to_frame, transform):
     def forward_transform(self):
         """
         Return the total forward transform - from input to output coordinate frame.
-
         """
-
-        if self._pipeline:
-            #return functools.reduce(lambda x, y: x | y, [step[1] for step in self._pipeline[: -1]])
-            return functools.reduce(lambda x, y: x | y, [step.transform for step in self._pipeline[:-1]])
-        else:
+        if not self._pipeline:
             return None
 
+        transform = functools.reduce(lambda x, y: x | y, [step.transform for step in self._pipeline[:-1]])
+
+        if self.bounding_box is not None:
+            # Currently compound models do not attempt to combine individual model
+            # bounding boxes. Get the forward transform and assign the bounding_box to it
+            # before evaluating it. The order Model.bounding_box is reversed.
+            transform.bounding_box = self.bounding_box
```

Comment:
Can you remind me why is it not necessary to reverse the bounding box when assigning to a transform? Is the default in modeling now `order=F`?

Reply:
https://github.com/spacetelescope/gwcs/blob/c01c6d0bf5a930dc36c9bcfb2c31531c1e591d71/gwcs/wcs.py#L1282-L1312

Returns a `astropy.modeling.bounding_box.ModelBoundingBox` not a tuple. That full object will be processed correctly by astropy.